modules/osdk-ansible-k8s-module-testing-locally.adoc

```diff
@@ -39,8 +39,8 @@ This example creates and deletes a namespace with the switch of a variable.
 +
 [source,yaml]
 ----
-- name: set test namespace to {{ state }}
-  k8s:
+- name: set test namespace to "{{ state }}"
+  community.kubernetes.k8s:
```

Comment:
We should install this `community.kubernetes` first, see: https://github.com/ansible-collections/community.kubernetes#installing-the-collection-from-ansible-galaxy
```console
$ ansible-galaxy collection install community.kubernetes
```
Otherwise, we will get the below errors:
```console
ERROR! couldn't resolve module/action 'community.kubernetes.k8s'. This often indicates a misspelling, missing collection, or incorrect module path.
```